Makefile

```
@@ -49,6 +49,17 @@ GO_OPERATOR_IMAGE_LDFLAGS="-X 'github.com/cilium/tetragon/pkg/version.Version=$(
 GOLANGCILINT_WANT_VERSION = 1.52.2
 GOLANGCILINT_VERSION = $(shell golangci-lint version 2>/dev/null)
 
+# Do a parallel build with multiple jobs, based on the number of CPUs online
+# in this system: 'make -j8' on a 8-CPU system, etc.
+#
+# (To override it, run 'make JOBS=1' and similar.)
+#
+ifeq ($(JOBS),)
+  JOBS := $(shell nproc)
+  ifeq ($(JOBS),0)
+    JOBS := 1
+  endif
+endif
```

```suggestion
JOBS ?= $(shell nproc)
```
I think you can simplify this a bit

---

true ;-)